### PR TITLE
Cli args for wizard

### PR DESCRIPTION
**Pull request by poteirard**
Hello!

Thanks for this great repo 🔥 

I just wanted to be able to create a job from my CI to create new projects.

I suppose the easiest way would be to enable CLI args in the CLI `wizard` so I just continued the work of #538 

Let me know if you need anything else

### CODE REVIEW

File: types/wizard.d.ts [hunk 1]
```diff
@@ -8,10 +8,13 @@ declare global {
   namespace LHCI {
     namespace WizardCommand {
       export interface Options {
-        wizard?: 'new-project' | 'reset-admin-token';
+        wizard?: 'new-project' | 'reset-admin-token' | 'reset-build-token';
```
Comment (poteirard) [inline types/wizard.d.ts:11]: I think this was missing in the type definition

**Comment by moshie** [comment]
@patrickhulce Just a heads up :)

File: docs/getting-started.md [hunk 2]
```diff
@@ -489,6 +489,18 @@ The _admin token_ is used to edit or delete data within the project. **KEEP THIS
 
 If your lhci server is behind basic auth, you can provide the username and password via the `--basicAuth.password` and `--basicAuth.username` command line flags.
 
+All these options are available for `lhci wizard`
```
Comment (patrickhulce) [inline docs/getting-started.md:492]: move and link to the configuration docs instead? don't want to have too many places where every flag is documented extensively :)

File: types/wizard.d.ts (hunk 1 above)
Comment (patrickhulce) [reply types/wizard.d.ts:11]: thanks 👍

File: packages/utils/src/api-client.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const btoa = typeof window === 'undefined' ? s => Buffer.from(s).toString('base6
 
 class ApiClient {
   /**
-   * @param {{rootURL: string, fetch?: import('isomorphic-fetch'), URL?: typeof import('url').URL, extraHeaders?: Record<string, string>, basicAuth?: LHCI.ServerCommand.Options['basicAuth']}} options
+   * @param {{projectExternalUrl?: string, serverBaseUrl?: string, basicAuth?: LHCI.ServerCommand.Options["basicAuth"], wizard?: "new-project" | "reset-admin-token" | "reset-build-token", extraHeaders?: Record<string, string>, storage?: LHCI.ServerCommand.StorageOptions, projectName?: string, projectBaseBranch?: string, rootURL: string}} options
```
Comment (patrickhulce) [inline packages/utils/src/api-client.js:17]: revert this? it does not accept these options or do anything with them

**Review by patrickhulce — CHANGES_REQUESTED** [review]
thanks @poteirard!

**Comment by connorjclark** [comment]
Hey @moshie, any chance you have time to address Patrick's feedback above?

**Pull request closed without merging**